lib/TwoFactorAuth.php

```diff
@@ -28,13 +28,13 @@ class TwoFactorAuth
     private static array $_base32lookup = array();
 
     public function __construct(
-        private ?string $issuer = null,
-        private int $digits = 6,
-        private int $period = 30,
-        private Algorithm $algorithm = Algorithm::Sha1,
-        private ?IQRCodeProvider $qrcodeprovider = null,
-        private ?IRNGProvider $rngprovider = null,
-        private ?ITimeProvider $timeprovider = null
+        private readonly ?string   $issuer = null,
+        private readonly int       $digits = 6,
+        private readonly int       $period = 30,
+        private readonly Algorithm $algorithm = Algorithm::Sha1,
+        private ?IQRCodeProvider   $qrcodeprovider = null,
+        private ?IRNGProvider      $rngprovider = null,
+        private ?ITimeProvider     $timeprovider = null
```

Comment:
potentially unnecessary readonly and potentially annoying spaces as previously discussed

Reply:
This is a PHP-CS-Fixer rule as well.